hdat/resultspec.py

```diff
@@ -68,3 +83,6 @@ def print_resultspec(result):
         return '{}/{}/{}'.format(suite_id, case_id, result_id)
     except KeyError:
         return 'Invalid result "{}"'.format(repr(result))
+    except TypeError:
+        print(result)
```

Comment:
Ooh good catch this is obsolete and will never show up since this was implemented in an earlier fix when result could return a string with an error message instead of a result.